torchbenchmark/util/framework/huggingface/model_factory.py

```diff
@@ -147,4 +147,7 @@ def eval(self) -> Tuple[torch.Tensor]:
             return (out["logits"], )
 
     def enable_amp(self):
-        self.amp_context = lambda: torch.cuda.amp.autocast(dtype=torch.float16)
+        if self.device == "cuda":
```

Comment:
can we move the `enable_amp` into model.py also?  cc: @xuzhao9

Reply:
Yeah, let's move `enable_amp` into `model.py`.